ui/src/components/inputs/InputsForm.vue

```diff
@@ -272,14 +271,8 @@
     import Markdown from "../layout/Markdown.vue";
     import Inputs from "../../utils/inputs";
     import DurationPicker from "./DurationPicker.vue";
-    import {inputsToFormData} from "../../utils/submitTask"
+    import {inputsToFormData} from "../../utils/submitTask";
 
-    import DeleteOutline from "vue-material-design-icons/DeleteOutline.vue";
```

Comment:
Why remove used icons? This creates warnings in the console?

<img width="689" height="762" alt="image" src="https://github.com/user-attachments/assets/ccab2ecd-a19a-4e4f-8d9e-8d7653f650c2" />